projects/skia/Dockerfile

```diff
@@ -34,56 +34,58 @@ WORKDIR $SRC/skia/third_party/externals/swiftshader/
 # TODO(metzman): Come up with a better long term solution, such as downloading
 # prebuilt libraries, than pinning swiftshader to a known working revision.
 RUN git checkout 45510ad8a77862c1ce2e33f0efed41544f5f048b
-WORKDIR $SRC/skia
+WORKDIR $SRC/skia_data
```

Comment:
I don't think you want to change the work dir.

Reply:
Yeah I would just undo that WORKDIR change.
Then I can land this.